wallet/src/sender.rs

```diff
@@ -92,8 +94,11 @@ fn build_send_tx(
 }
 
 pub fn issue_burn_tx(config: &WalletConfig, keychain: &Keychain, amount: u64) -> Result<(), Error> {
+	let keychain = &Keychain::burn_enabled(keychain, &Identifier::zero());
```

Comment:
Pass in the Identifier here for flexibility.

Reply:
We could potentially generalize this further and pass in multiple Identifiers.
I think we'd need to randomly generate secret keys at that point (and not rely on a single known value).
Maybe this is a subsequent PR.